src/Lachain.Core/RPC/HTTP/Web3/AccountServiceWeb3.cs

```diff
@@ -170,10 +183,32 @@ public JArray GetAccounts()
         [JsonRpcMethod("eth_sign")]
         private string Sign(string address, string message)
         {
-            // TODO: implement message signing
-            //var addressUint160 = address.HexToUInt160();
-            return Web3DataFormatUtils.Web3Data("".HexToBytes());
-            //throw new ApplicationException("Not implemented yet");
+            if (address is null)
+            {
+                throw new ArgumentException("address should not be null");
+            }
+
+            byte[]? messageBytes = Encoding.ASCII.GetBytes(_messagePrefix + message.Length + message);
```

Comment:
No,  it is difficult to verify this signature on the client side, Use `Encoding.ASCII.GetBytes` for conversion of the  `_messagePrefix + message.Length` and `HexToBytes` for conversion of the `message`,  after that concatenate 2 ByteArrays